tests/ManageList.test.jsx

```diff
@@ -2,24 +2,6 @@ import { render, screen } from '@testing-library/react';
 import { ManageList } from '../src/views/ManageList';
 import { MemoryRouter } from 'react-router-dom';
 
-beforeEach(() => {
-	Object.defineProperty(window, 'localStorage', {
-		value: {
-			getItem: vi.fn((key) => {
-				if (key === 'tcl-shopping-list-path') {
-					return '/groceries';
-				}
-				return null;
-			}),
-			setItem: vi.fn(),
-			clear: vi.fn(),
-		},
-		writable: true,
-	});
-
-	vi.spyOn(window, 'alert').mockImplementation(() => {});
-});
-
```

Comment:
Same here,  you only have toe delete line 20. You're removing the mocked local storage value to be able to simulate the groceries list, which is why all the tests are failing